docs/RUNNING_LOCALLY_DOCKER.md

```diff
@@ -8,17 +8,35 @@ cp app/app/local.env app/app/.env
 
 ## Startup server
 
+*Check that you have Docker and Docker-compose properly installed*
+```shell
+docker --version
+Docker version 18.09.7, build 2d0083d
+
+docker-compose --version
+docker-compose version 1.24.1, build 4667896b
+```
+The above should work.
+
 ### Running in Detached mode
 
 ```shell
 docker-compose up -d --build
 ```
 
+The above would create a background daemon when it finished installation. It takes pretty long time, an hour or more. The good news is that it runs on its own. If you have error while running this it is likely to come from docker and docker-compose.
+
 ### Running in the foreground
 
 ```shell
 docker-compose up --build
 ```
+### Screens during building
+
+![alt text](https://raw.githubusercontent.com/janus/web/janus_doc/docs/imgs/screenshot1.png "Installation screenshot")
```

Comment:
Can you please include these files in this repo instead of linking it from your own? thanks!

Reply:
Thanks. I have moved the files to repo.